rest/route/user_test.go

```diff
@@ -158,3 +160,95 @@ func (s *UserRouteSuite) TestSaveFeedback() {
 	s.NotEqual(time.Time{}, feedback[0].SubmittedAt)
 	s.Len(feedback[0].Questions, 1)
 }
+
+func TestPostUserPermissions(t *testing.T) {
+	assert := assert.New(t)
+	assert.NoError(db.ClearCollections(user.Collection, evergreen.ScopeCollection, evergreen.RoleCollection))
+	env := evergreen.GetEnvironment()
+	_ = env.DB().RunCommand(nil, map[string]string{"create": evergreen.ScopeCollection})
+	u := user.DBUser{
+		Id: "user",
+	}
+	assert.NoError(u.Insert())
+	ctx := context.Background()
+	handler := makeModifyUserPermissions(&data.DBConnector{})
+
+	// no user should return the appropriate error
+	invalidBody := `{ "foo": "bar" }`
+	request, err := http.NewRequest(http.MethodPost, "", bytes.NewBuffer([]byte(invalidBody)))
+	assert.NoError(err)
```

Comment:
this should probable be a require because otherwise later things are very likely to panic.

Reply:
changed the setup things to requires